From ab1e6ca291c467953f5a4b308595a4287618b836 Mon Sep 17 00:00:00 2001
From: Elena Zannoni <ezannoni@redhat.com>
Date: Fri, 27 Oct 2017 21:07:50 +0200
Subject: gdb.gdb/selftest.exp: Add matching on specific Red Hat only version
 string

FileName: gdb-6.3-rh-testversion-20041202.patch

;; Match the Fedora's version info.
;;=fedora

2003-02-24  Elena Zannoni  <ezannoni@redhat.com>

        * gdb.gdb/selftest.exp: Add matching on specific Red Hat only version
        string.
---
 gdb/testsuite/gdb.gdb/selftest.exp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gdb/testsuite/gdb.gdb/selftest.exp b/gdb/testsuite/gdb.gdb/selftest.exp
index 712060c9ae..a4a446f9df 100644
--- a/gdb/testsuite/gdb.gdb/selftest.exp
+++ b/gdb/testsuite/gdb.gdb/selftest.exp
@@ -53,6 +53,9 @@ proc test_with_self { } {
 	-re ".\[0-9\]+ = +.+ +0x.*\[0-9.\]+.*$gdb_prompt $" {
 	    pass "printed version with cast"
 	}
+	-re ".\[0-9\]+ = .(Fedora|Red Hat Enterprise Linux) \[\\(\\)0-9.a-z\\-\]+.*$gdb_prompt $" {
+	    pass "printed version Fedora or Red Hat Enterprise Linux only"
+	}
     }
 
     # start the "xgdb" process
-- 
2.14.3

